src/components/general/Pill.module.css

```diff
@@ -26,7 +26,7 @@
 }
 
 .pill .text {
-  color: #A19EB0;
+  color: var(--neutral-light-3);
```

Comment:
assume this was cool w/ julian for non matrix pill right?

Reply:
yeah if you look at the figma designs (inc non-matrix), the prod value was wrong